### PR TITLE
automate device enrollment key rotation in google core iot

### PR DESCRIPTION
**Pull request by eugenosm**
closes  https://github.com/WoTTsecurity/api/issues/278

### CODE REVIEW

File: docs/examples/google-core-iot/README.md [hunk 1]
```diff
@@ -141,3 +141,36 @@ Received message 'b'Hello world'' on topic '/devices/x.d.wott.local/commands' wi
...
+
+## Automatic enrolling/updating script 
+
+If you have a fleet of devices, it is difficult to perform the above steps from time to time to renew the certificates or upload new ones to just created devices.
+It would be right to have a script that allows you to perform all these tasks, as well as register new devices in the Google IoT registry, according to the list of devices in WoTT Dash.
+The following example script allows you to do this. This works in the following scenario.
+
+- Retrieve all devices from WoTT Dash
+- Retrieve all devices from Google IoT Registry.
+- Check all" Google devices " for updated certificates in the WoTT list. And update if so.
```
Comment (rptrchv) [inline docs/examples/google-core-iot/README.md]: Sounds weird

**Comment by vpetersson** [comment]
@fshmcallister This is relevant for you.

File: docs/examples/google-core-iot/README.md (hunk 1 above)
Comment (eugenosm) [reply docs/examples/google-core-iot/README.md]: Changed that part a bit.

Comment (fshmcallister) [reply docs/examples/google-core-iot/README.md]: Don't worry about wording, I can touch that up later.

**Comment by vpetersson** [comment]
@eugenosm this should actually be moved [to this repo](https://github.com/WoTTsecurity/google-core-iot).

**Comment by eugenosm** [comment]
moved to new repo
https://github.com/WoTTsecurity/google-core-iot/pull/2

**Pull request closed without merging**